### ISSUE TITLE
Sprint 4 – Scrum Meeting (20-12-2016)

### ISSUE THREAD

**Issue by lkc442012** [mewind/RSDSupermarketPOS#22]
Good afternoon, everyone! May I know how is the progression today?

**Comment by huylin25** [comment]
Hmm, i have start doing the sales report generation part. Now i am using the print function to display the report following the standard or format that we want.

**Comment by lkc442012** [comment]
Well done! What is your next step to further develop it?

**Comment by huylin25** [comment]
After formating all the header and body of the sales report, I plan to connect the report to the database to display the information such as product name, price, quantity and total sales.

**Comment by mewind** [comment]
So you are going to use SQL command to list out which information to be displayed on the report?

**Comment by huylin25** [comment]
Yes, I will display 3 monthly sales report which includes June, July and August in the report page. For the June Sales Report, I will try to display the relevant information for all the transactions that is carried out between 01/06/2016 and 30/06/2016 using SQL command. If I success, I will do the same to the others report.

**Comment by lkc442012** [comment]
Okay, do you facing any difficulties?

**Comment by huylin25** [comment]
I just not very sure about the SQL format and spent a lot of time on research about it. Maybe I need some time to complete it.

**Comment by juzplayplay** [comment]
Huy Lin, I will try to search for the SQL format since we are doing the report generation. So maybe I will help you to solve your problem first, then I continue my part on tomorrow.

**Comment by mewind** [comment]
Alright, if Huy Lin's part is solved, then the exception and summary report will not be too difficult.

**Comment by huylin25** [comment]
Okay, thank you very much!

**Comment by lkc442012** [comment]
If you need help, we can help you to find the solution together.

**Comment by mewind** [comment]
Yea.

**Comment by lkc442012** [comment]
So today's meeting is done, thank you everyone! Keep on going!

**Comment by juzplayplay** [comment]
Sure!
